test/session.test.js

```diff
@@ -758,3 +761,71 @@ test('only save session when it changes', async t => {
   // no set-cookie
   t.is(setCookieHeader2, undefined)
 })
+
+test('when unsignSignedCookie is true sessions should still be managed correctly', async t => {
+  t.plan(12)
```

Comment:
because you're not doing any assertions in callbacks that you need to wait for in order to consider the test complete, you can remove the .plan here